docs/sources/alerting/unified-alerting/opt-in.md

```diff
@@ -6,11 +6,11 @@ weight = 128
 
 # Enable Grafana 8 Alerts
```

Comment:
Should we remove this section and simply add a note that you can optionally disable Grafana 8 alerts and use legacy alerts?